src/modules/requirements/DayCyclePartRequirement.ts

```diff
@@ -0,0 +1,21 @@
+import { AchievementOption } from '../GameConstants';
+import Requirement from './Requirement';
+import DayCyclePart from '../dayCycle/DayCyclePart';
+import DayCycle from '../dayCycle/DayCycle';
+
+export default class DayCyclePartRequirement extends Requirement {
+    private updateTrigger = ko.observable(0);
+    constructor(public dayCycleParts: DayCyclePart[], option = AchievementOption.more) {
+        super(1, option);
+    }
+
+    public getProgress(): number {
+        this.updateTrigger();
+
+        return Number(this.dayCycleParts.includes(DayCycle.currentDayCyclePart()));
+    }
```

Comment:
I checked everything twice looking for one last thing to complain about, and I got a bite:

`updateTrigger` isn't needed, you got the setInterval but these slipped through
```suggestion
    constructor(public dayCycleParts: DayCyclePart[], option = AchievementOption.more) {
        super(1, option);
    }

    public getProgress(): number {
        return Number(this.dayCycleParts.includes(DayCycle.currentDayCyclePart()));
    }
```

Reply:
oops right, let's try that "commit suggestion" button